### DISCUSSION TITLE
如何修改进度条的样式

### DISCUSSION THREAD

**Discussion by izumiChan16** [hooke007/mpv_PlayKit#500] [Q&A 问答]
我尝试修改进度条的高度和透明度，因为鼠标放在进度条上，进度条颜色过深会挡住字幕

修改了 `\portable_config\script-opts.conf` 中的 `script-opts-append = osc-seekrangealpha=50`无果
修改了 `\portable_config\mpv.conf` 中的 `osd-bar`相关参数依旧没有变化

我期望的效果是，进度条在显现的时候高度可以降低，并且透明度降低

版本：mpv-lazy-20240824.exe 标准版本

求助于各位，希望知道该如何修改，才能进行有效的改动

**Comment by hooke007** [comment]
lazy甚至都不使用osc，不知道你调一个无关脚本的作用是什么。。。

**Comment by hooke007** [comment]
去看uosc的对应选项
